src/contents/_activity_stream.scss

```diff
@@ -73,7 +73,8 @@
         transition: border-color 0.5s var(--animation-easing-function), box-shadow 1s var(--animation-easing-function);
       }
       @include Option("userContent.page.field_border") {
-        .search-wrapper .search-inner-wrapper:hover input {
+        .search-wrapper .search-inner-wrapper:hover input,
+        .search-inner-wrapper:hover button {
```

Comment:
Thank you again for your contribution.

As far as I know, `.search-inner-wrapper:hover button` is submit button.
It works well with `.search-inner-wrapper:hover input` alone.
Would you like to check again?

![image](https://user-images.githubusercontent.com/25581533/208249599-a9788bea-b47d-4053-ae0c-7883469f5dc7.png)

Reply:
The hover highlight will work fine if you use the older searchbar (`browser.newtabpage.activity-stream.improvesearch.handoffToAwesomebar` set to `false`), but it does not work with the new "fake" searchbar (that key when set to `true`).

Tho this patch is quite hacky and it may need a rework (so that it won't conflict with the enter/submit button)

Reply:
I got it :+1: